servantes/web/templates/index.tpl

```diff
@@ -2,28 +2,131 @@
+  }
   
-  <div style="display: grid; width: 80vw; height: 50vw; margin: auto; grid-template-columns: repeat(3, 1fr); grid-gap: 0.5em; grid-template-rows: repeat(3, 1fr);">
-    {{range $i, $service := .Services}}
-      <div style="position:relative;box-shadow: inset 0 0 4em #00bfff;padding:2em;">
-        <div style="position:absolute; font-size:0.8em; bottom:0.5em; right:0.5em; z-index:1; text-align:right;">
-          service: {{$service.Name}}<br>
-          status: {{$service.Status}}<br>
-          restartCount: {{$service.RestartCount}}<br>
-          age: {{$service.HumanAge}}
-        </div>
-        <iframe frameborder="0"
-                style="overflow:hidden;box-sizing:content-box;width:100%;height:100%;" src="/s/{{$service.Name}}"></iframe>
-      </div>
-    {{end}}
-      
-  </div>
+  html {
+    margin: 0;
+  }
+
+  body {
+    margin: 0;
+    font-family: 'Varela Round', sans-serif;
+    color: var(--dark-gray);
+    background-color: var(--white-var1);
+  }
+
+  header {
+    font-family: 'Lora', serif;
+    color: var(--tan-dark);
+    margin-top: 0;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 0;
+    max-width: 100em;
+    padding-top: 1em;
+    padding-left: 2em;
+    padding-right: 2em;
+    font-size: 2em;
+    text-align: right;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
+    list-style-type: none;
+    padding: 0;
+  }
+
+  ul.services {
+    display: grid; 
+    width: 100vw; 
+    max-width: 100em;
+    margin-left: auto;
+    margin-right: auto;
+    padding: 2em;
+    grid-gap: 2em; 
+    grid-template-columns: repeat(3, 1fr); 
+    box-sizing: border-box;
+  }
+
+  li.service-item {
+    background-color: var(--white);
+    padding: 0;
+    position: relative;
+    min-height: 15em;
+    border-right: 10px solid var(--tan);
+    border-bottom: 10px solid var(--tan);
+    /* Make room for K8s Data */
+    margin-top: 2.5em; 
+  }
+
+  ul.k8s-data {
+    color: var(--white-var1);
+    background-color: var(--dark-gray);
+    position: absolute;
+    font-size: 0.8em; 
+    top: -2.5em;
+    height: 2.5em;
+    z-index: 1; 
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 1em;
+    padding-right: 1em;
+    border-top-left-radius: 0.5em;
+    border-top-right-radius: 0.5em;
+    /* Align with border below: */
+    left: -1px;
+    right: -1px;
+  }
+
+  ul.k8s-data span + span {
+    padding-left: 0.5em;
+  }
 
+  iframe {
+    overflow: hidden;
+    box-sizing: content-box;
+    border: 1px solid var(--dark-gray);
+    width: 100%;
+    height: 100%;
+  }
+</style>
+<body>
+  <main>
+    <header>Servantes ✴︎</header>
+    <ul class="services">
+      {{range $i, $service := .Services}}
+        <li class="service-item">
+          <ul class="k8s-data">
+            <li>{{$service.Name}} — {{$service.Status}}</li>
+            <li>
+              <span>restartCount: {{$service.HumanAge}}</span>
```

Comment:
HumanAge -> RestartCount

i think this got messed up on merge :)

Reply:
Wait- you don't `sudo reboot` every year on your birthday?